src/interface/IEbtcZapRouter.sol

```diff
@@ -9,6 +9,21 @@ interface IEbtcZapRouter {
         bytes32 s;
     }
 
+    enum EthVariantZapOperationType {
+        OpenCdp,
+        AdjustCdp,
+        CloseCdp
+    }
+
+    event ZapOperationEthVariant(
+        bytes32 indexed cdpId,
+        EthVariantZapOperationType indexed operation,
+        bool isCollateralIncrease,
+        address indexed collateralToken,
+        uint256 collateralTokenDelta,
+        uint256 stEthDelta
```

Comment:
not sure about the naming here. In `EbtcZapRouter.sol`, it seems most function parameter use the prefix `coll` to refer the collateral in CDP but here it seems to choose another prefix `stETH`